bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java

```diff
@@ -24,6 +24,7 @@
 import org.eclipse.smarthome.core.library.types.IncreaseDecreaseType;
```

Comment:
Please move all the changes on this class in a different PR because they have no link with the current feature added.
If you want to push this changes, I will suggest thjat you consider the hue light handler too because the code is very similar.
It makes no real sense to make it different in one place only.

Reply:
The scene channel is added to groups here. When navigting through the class, I think these are quite minor changes that improved the readability of the code greatly for me. Is it of importance that the two classes are kept similar (except for the case that someone want to refactor both class)?

Reply:
This will not help the maintenance if this is coded differently in thge different classes.